src/test/java/com/deque/axe/android/AxeTest.java

```diff
@@ -280,6 +285,139 @@ public void testAddRule() {
     assertEquals(axe.axeConf.ruleInstances().size(), 10);
   }
 
+  @Test
+  public void jsonAxeRuleResultTestSpecs() throws IOException {
```

Comment:
BLOCKER: Document the importance and purpose of this test case. ESPECIALLY JTBD.

Reply:
@chriscm2006 I don't know why the code snippet above is not showing the updated code because I have added a `Javadoc` on the top of this test. The changes are visible in File Changes.